unittest.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jun 26 13:53:41 2017
+
+@author: siddhant
+"""
+
+import unittest
+from point import Point
+import constants as c
+from line import Line
+
+
+p1 = Point(1.0, 2.0, 3.0)
+p2 = Point(2.0, 4.0, 6.0)
+l1 = Line(p1, p2)
+p3 = Point(1.0, 2.0, 3.0)
+p4 = Point(6.0, 12.0, 18.0)
+l2 = Line(p3, p4)
+
+class PointTestCase (unittest.TestCase):
+   
```

Comment:
I realized that there should be tests for the comparison operators (==, !=, <, etc.) that also test for the false case. Right now most of the tests would pass if the Point functions were changed to just return True.